Cargo.toml

```diff
@@ -13,3 +13,21 @@ members = [
 exclude = [
     "fluent-cli",
 ]
+
+[workspace.dependencies]
+criterion = "0.3"
+fluent-langneg = "0.13"
+futures = "0.3"
+iai = "0.1"
+intl_pluralrules = "7.0.1"
+rustc-hash = "1"
+serde = "1.0"
+serde_json = "1.0"
+thiserror = "1.0"
+tokio = "1.0"
+unic-langid = "0.9"
+
+fluent-bundle = { version = "0.15.2", path = "fluent-bundle" }
```

Comment:
Question: Do you know if this works without the version, or if `cargo-release` automatically updates this version number? It would be nice not to have to maintain multiple sets of version numbers, but I'm unsure how this works.

Reply:
I just made a quick test with `cargo-release` and the main `Cargo.toml` is indeed updated. But it probably doesn't make much sense to specify the version there so I'll clean that ^^